src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,31 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+type Props = {
+  todo: Todo | null;
+  onClose: () => void;
+};
+export const TodoModal: React.FC<Props> = ({ todo: todo, onClose }) => {
```

Comment:
The prop `todo` is correctly named here. Ensure that this aligns with the rest of the application, especially in the `App.tsx` file where the `TodoModal` is used.